Documentation/install/upgrade.rst

```diff
@@ -162,6 +164,8 @@ Kubernetes resources are updated accordingly to version you are upgrading to:
     .. parsed-literal::
 
       helm template |CHART_RELEASE| \\
+        --set config.upgradeCompatibility=1.7 \\
+        --agent.keepDeprecatedProbes=true \\
```

Comment:
Is this a leftover? `--agent.keepDeprecatedProbes=true` will not make sense for users upgrading from 1.8 to 1.9. Actually, shouldn't these instructions be version agnostic as we have dedicated sections when users are upgrading to a specific version?

Reply:
It's a hard question, it will still be needed for most users as they come from  1.7 -> 1.8 -> 1.9. 

The alternative is to default on or HTTP probes but also provide an options and to default off for <1.8 upgradeCompatibility but allow users to overwrite it.